guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-capsule-server.adoc

```diff
@@ -22,7 +22,7 @@ For more information, see {InstallingSmartProxyDocURL}installing-capsule-server-
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # katello-certs-check \
--c __/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem__ \      <1>
+-t {smart-proxy-context} -c __/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem__ \      <1>
```

Comment:
In upstream `smart-proxy-context` is `smart-proxy`. I looked at it and the help text suggests (in upstream) it's either `katello` or `foreman-proxy` but the code says that anything other than `katello` will trigger the non-server code. So `smart-proxy` and `foreman-proxy` will be handled the same.

The only thing I can think of is that a user may run `katello-certs-check -h` and be confused.

Turns out there is already a variable for this:
```suggestion
-t {certs-proxy-context} -c __/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem__ \      <1>
```

See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_creating-a-custom-certificate.adoc as well:

> If you set a wildcard value, you must add the `-t {certs-proxy-context}` option when you use the `katello-certs-check` command.

This may be a bad example since that particular line is plain wrong. Given the `ifeval` it should state to **always** pass `-t {certs-proxy-context}` instead of only when a wildcard value is set, just like in this change.

Reply:
Thanks so much, @ekohl